package.json

```diff
@@ -18,12 +18,14 @@
     "dist/*"
   ],
   "main": "dist/_hyperscript.min.js",
+  "types": "dist/_hyperscript.d.ts",
```

Comment:
https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html#editing-the-packagejson